src/index.js

```diff
@@ -118,11 +118,25 @@ const clientHandler = async (taiko, eventHandler) => {
   });
 };
 
+const slowdownVideo = async (videoPath, outputDir, fps) => {
+  const args = [
+    '-y',
+    '-i', videoPath,
+    '-vf', 'setpts=N/(' + fps + '*TB)',
+    outputDir + videoPath.substring(videoPath.lastIndexOf('/'))
+  ];
+
+  const proc = spawn('ffmpeg', args);
+
+  await once(proc, 'close');
+}
+
```

Comment:
**suggestion:** Would it make more sense to just allow the fps be an option that is directly provided into `startRecording` rather than having to call a separate function to slow it down? Couple of other potential issues with this approach are: 1) there is a reliance on having the input screenshots being ready by the time this function is called. 2) it creates another extra video and I'm thinking that one properly timed video is all that's needed rather having two in two different speeds (my assumption).

Reply:
in addition to moving it to `start`, it would be nice to have a reasonable default `fps` set so user's don't have to explicitly specify it. You probably have a good idea of what that number should be.